From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: TheScarastic <warabhishek@e.email>
Date: Tue, 4 Apr 2023 11:21:27 +0000
Subject: [PATCH 2/4] ThemePicker: Adjust permission for changed packagename

Change-Id: If81882ee61b9bcc652ff3c73f2507867c3d5a5a0
[a17: keep the com.android.launcher3 permissions as well, because /e/OS's
 a17 BlissLauncher bring-up branch ships as com.android.launcher3 for now]
(cherry picked from commit f25b9a932f87381576432c272f663a67845a6874)
---
 AndroidManifest.xml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 3010f428..b60288a1 100755
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -26,6 +26,7 @@
     <queries>
         <package android:name="android"/>
         <package android:name="com.android.launcher3"/>
+        <package android:name="foundation.e.blisslauncher"/>
         <package android:name="com.android.settings"/>
         <package android:name="com.android.systemui"/>
         <!-- Package of camera for icon shape -->
@@ -66,6 +67,8 @@
 
     <uses-permission android:name="com.android.launcher3.permission.READ_SETTINGS" />
     <uses-permission android:name="com.android.launcher3.permission.WRITE_SETTINGS" />
+    <uses-permission android:name="foundation.e.blisslauncher.permission.READ_SETTINGS" />
+    <uses-permission android:name="foundation.e.blisslauncher.permission.WRITE_SETTINGS" />
 
     <application
         tools:replace="android:icon,android:name"
